holmes/plugins/sources/prometheus/plugin.py

```diff
@@ -72,6 +74,22 @@ def fetch_issues(self) -> List[Issue]:
             )
             for alert in alerts
         ]
+        return self.label_filter_issues(issues)
```

Comment:
Is there a reason we filter on alert name above but on alert labels below?

Lets be consistent unless there is a very good reason not to be.